sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -257,12 +257,14 @@ class FindDataSourceTable(sparkSession: SparkSession) extends Rule[LogicalPlan]
       QualifiedTableName(table.identifier.catalog.get, table.database, table.identifier.table)
     val catalog = sparkSession.sessionState.catalog
     val dsOptions = DataSourceUtils.generateDatasourceOptions(extraOptions, table)
+    val readDataSourceCacheIgnoreOptions: Boolean =
+      SQLConf.get.getConf(SQLConf.READ_DATA_SOURCE_CACHE_IGNORE_OPTIONS)
     catalog.getCachedTable(qualifiedTableName) match {
       case null =>
         val dataSource =
           DataSource(
             sparkSession,
-            // In older version(prior to 2.1) of Spark, the table schema can be empty and should be
+            // In older version(prior to 2.1) of Spark,the table schema can be empty and should be
```

Comment:
The space can be put back between the words?  To me, its harder to read without it?